metatron-app/metatron-discovery
# #3569 separate session timeout and other ip access check

### Description

separate session timeout and other ip access check

**Related Issue** : #3569 



### How Has This Been Tested?

1. Set session timeout and check ip
**Set the values ​​of the ACCESS_TOKEN_VALIDITY and REFRESH_TOKEN_VALIDITY columns the same.**
```
URL : /api/oauth/polaris_client
Method : POST
RequestBody : example
{
    "checkIp": true,
    "accessTokenValiditySecond": 60,
    "refreshTokenValiditySecond": 60
}
Header : Authorization (Bearer {})
```
2. Login metatron Discovery.
3. Check if the timeout time set in step 1 is applied.
4. Prepare two different test IPs.
5. Log in as admin from Client A
6. Login as admin from Client B
7. Verify that Client A logs out with an authentication expiration message when navigating to another page
8. Client B should work normally when moving the page
9. Set session timeout and disable other access IP blocking.
```
URL : /api/oauth/polaris_client
Method : POST
RequestBody : example
{
    "checkIp": false
}
Header : Authorization (Bearer {})
```
10. Repeat steps 2 through 6.
11. Both Client A and B should work normally when moving the page

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
